README.md

```diff
@@ -5,23 +5,23 @@ This is licensed under MIT so please feel free to fork and use as you'd like.
 
 If you like it, please consider leaving a github star.  I think that it would be super cool to see how many people this helps.
 
-## the autoresponse
+## the autoresponse asking for details
 
 ```
-Thanks so much for reaching out. I'm always interested in hearing about what new and exciting opportunities are out there. As a software engineer, I'm sure you can imagine that I get a very high volume of recruiters reaching out on LinkedIn or by email. It is a wonderful position of privilege to be in and I'm thankful for it.
+Thanks so much for reaching out. I'm always interested in hearing about what new and exciting opportunities are out there. As a software engineer, I'm sure you can imagine that I get a very high volume of recruiters reaching out on LinkedIn or by email. It is a wonderful position of privilege to be in, and I'm thankful for it.
 
-It does however mean that I don't have the time to hop on a call with everyone who reaches out. A lot of the time, incoming messages represent a very poor fit indeed.
+It does, however, mean that I don't have the time to hop on a call with everyone who reaches out. A lot of the time, incoming messages represent a very poor fit indeed.
 
-I would love to continue the conversation, but before I do, I'd like to level set around the level of seniority that you're looking for. 
+I would love to continue the conversation, but before I do, I'd like to know more about the level of seniority that you're looking for
 
-Can you send along a job description, remote info, interview process and total compensation details for the role? 
```

Comment:
I think keeping the part about the interview process would be helpful

Reply:
Could you check if it looks better now? Thanks!